src/transformers/models/aria/configuration_aria.py

```diff
@@ -0,0 +1,175 @@
+#                🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
+#           This file was automatically generated from src/transformers/models/aria/modular_aria.py.
+#               Do NOT edit this file manually as any edits will be overwritten by the generation of
+#             the file from the modular. If any change should be done, please apply the change to the
+#                          modular_aria.py file directly. One of our CI enforces this.
+#                🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
+
+
+from ...configuration_utils import PretrainedConfig
+from ...modeling_rope_utils import rope_config_validation
+from ..auto import CONFIG_MAPPING
+
+
+class AriaTextConfig(PretrainedConfig):
+    """
+    Configuration class for Aria language model.
+
+    This class extends the LlamaConfig to include additional parameters specific to the Mixture of Experts (MoE) architecture.
+
+    Args:
+        moe_intermediate_size (`int`): The intermediate size for MoE layers. Default is 4096.
+        moe_num_experts (int): The number of experts in the MoE layer. Default is 8.
+        moe_topk (int): The number of top experts to route to for each token. Default is 2.
+        moe_z_loss_coeff (float): The coefficient for the auxiliary z-loss. Default is 1e-5.
+        moe_aux_loss_coeff (float): The coefficient for the auxiliary load balancing loss. Default is 1e-3.
+        moe_num_shared_experts (int): The number of shared experts. Default is 2.
+        **kwargs: Additional keyword arguments to be passed to the parent LlamaConfig.
```

Comment:
i think docstring should follow the standards, from https://github.com/huggingface/transformers/tree/main/docs#defining-arguments-in-a-method. Same for the `AriaConfig`. Also would be nice to add some usage examples [as in LLaVA](https://github.com/huggingface/transformers/blob/ae49b218c3d718df90d8e4a109016450fb8f0632/src/transformers/models/llava/configuration_llava.py#L58-L77)

For ex, adding the `*optional*, defaults to ....` at the end